ddl/ddl_test.go

```diff
@@ -98,8 +88,13 @@ func (d *ddl) restartWorkers(ctx context.Context) {
 	}
 }
 
+// TestLeakCheckCnt is the check count in the pacakge of ddl.
+// In this package CustomParallelSuiteFlag is true, so we need to increase check count.
```

Comment:
Need to enable CustomParallelSuiteFlag and split tests in ddl_test.go?

Reply:
We only need to set `CustomParallelSuiteFlag` in TestT.